.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
+root = true
+
+[*.{yml,yaml,dart,json,gitignore}]
```

Comment:
Dartfmt already exists. Unnecessary.

Reply:
Maybe create a dartfmt config?

Reply:
On it.

Reply:
There is a way to auto-format Dart code with the VS Code setting `editor.formatOnSave` when the Flutter extension is installed. However every editor would need to install it, and it only works for Dart files. [More info on this here.](https://flutter.dev/docs/development/tools/formatting)

We should keep .editorconfig so that it would trim whitespace and add a newline on Dart files as well. It doesn't hurt - the Dart formatter works fine alongside Editorconfig - and it will also trim the file if users don't have the setting enabled, which yields better Git diffs.

Reply:
As far as I am aware, there is no way to add a dartfmt config file. However, I can add a vscode setting to enable it for us.